apigee_edge.module

```diff
@@ -1459,3 +1458,11 @@ function apigee_edge_get_my_developer_apps_title(?UserInterface $user = NULL): T
 
   return $title;
 }
+
+/**
+ * Implements hook_cache_flush().
+ */
+function apigee_edge_cache_flush() {
+  // Clear token data.
+  \Drupal::service('apigee_edge.authentication.oauth_token_storage')->saveToken([]);
```

Comment:
It's a cache item that is auto generated.

Reply:
I would not consider access token as a "cache item". Why we want to enforce re-authentication to Apigee Edge with username/pwd/client_id/client_secret on cache clear? Because this is what we achieve with this.

@cnovak What do you think?

Reply:
It's can be used for sanitization or making sure the SDK client re-authenticates. There isn't much of a downside as far as I see it. Forcing Apigee Edge to re-authenticate is part of the point.

Reply:
I don't see much of a downside of using the refresh as a way to clear out the tokens, it may be the easiest approach to support the module if we find out any token errors are happening.

Reply:
I do agree that we should provide a way to flush out access token/refresh token the system for debug purposes, but probably we should have a dedicated button for that in the client configuration UI instead. Cache clear could happen multiple times when update hooks are running in Drupal (not just our module's update hooks) and my concern is that we can run into some issues if we send too many re-authentication requests to Apigee Edge when Drupal updates are running.
It is easy to add this later if we decide to do it this way, but it also works for me if it leave it here until the next release when we re-consider whether this is the right way to solve this.